README.md
# Awesome critique of crypto/web3

Awesome critique of crypto/web3, etc. Contributions are welcome.

## Critique

### General

* [The problem with NFTs](https://www.youtube.com/watch?v=YQ_xWvX1n9g) - 2022-01-21 - by Dan Olson (Documentary) 📺 [👉 Highly recommended 👈]
  * [Three things Web3 should fix in 2022](https://www.theverge.com/2022/1/28/22906010/web3-nft-internet-history-video-platformer) a response to The Problem with NFTs - 28 Jan 2022
* Stephen Diehl series - https://www.stephendiehl.com/blog.html
  * [The Case Against Crypto](https://www.stephendiehl.com/blog/against-crypto.html) - December 31, 2021
  - [Blockchainism](https://www.stephendiehl.com/blog/blockchainism.html) - December 11, 2021
  - [Web3 is Bullshit](https://www.stephendiehl.com/blog/web3-bullshit.html) - December 4, 2021
  - [The Internet's Casino Boats](https://www.stephendiehl.com/blog/casino-boats.html) - December 1, 2021
  - [The Token Disconnect](https://www.stephendiehl.com/blog/disconnect.html) - November 27, 2021
  - [The Handwavy Technobabble Nothingburger](https://www.stephendiehl.com/blog/nothing-burger.html) - November 24, 2021
  - [Ice-Nine for Markets](https://www.stephendiehl.com/blog/ice-nine.html) - November 23, 2021
  - [The Tinkerbell Griftopia](https://www.stephendiehl.com/blog/tinkerbell.html) - November 19, 2021
  - [Decentralized Woo Hoo](https://www.stephendiehl.com/blog/decentralized-woo.html) - November 16, 2021
  - [The Intellectual Incoherence of Cryptoassets](https://www.stephendiehl.com/blog/crypto-absurd.html) - November 7, 2021
  - [On Unintentional Scams](https://www.stephendiehl.com/blog/crypto-scams.html) - July 23, 2021
  - [How to Destroy Bitcoin](https://www.stephendiehl.com/blog/destroy-bitcoin.html) - July 13, 2021
  - [The Non-Innovation of Cryptocurrency](https://www.stephendiehl.com/blog/non-innovation.html) - July 7, 2021
  - [The Oncoming Ransomware Storm](https://www.stephendiehl.com/blog/ransomware.html) - May 11, 2021
  - [Et tu, Signal?](https://www.stephendiehl.com/blog/signal.html) - April 7, 2021
  - [The Political Case for a Blanket Cryptocurrency Ban](https://www.stephendiehl.com/blog/banbitcoin.html) - March 30, 2021
  - [Bitcoin: The Postmodern Ponzi](https://www.stephendiehl.com/blog/ponzi.html) - February 27, 2021
  - [The Crypto Chernobyl](https://www.stephendiehl.com/blog/chernobyl.html) - February 10, 2021
  - [Gamestop, Bitcoin and the Commoditization of Populist Rage](https://www.stephendiehl.com/blog/gamestop.html) - February 3, 2021
* [Today on Sick Sad World: How The Cryptobros Have Fallen](https://www.jwz.org/blog/2022/01/today-on-sick-sad-world-how-the-cryptobros-have-fallen/) - 2022-01-04 by Jamie Zawinski (legendary coder, co-founder of Mozilla etc.)
* [Web3 First Impressions](https://moxie.org/2022/01/07/web3-first-impressions.html) - 2022-01-07 Moxie Marlinspike, co-founder of Signal etc.
* [Bitcoin, Currencies, and Fragility by Nassim Taleb - 27 Jun 2021](https://arxiv.org/abs/2106.14204) - highly critical paper by author Black Swan etc.
* https://watershed.co.uk/studio/news/2021/12/03/case-against-crypto
* [The European Money and Finance Forum - The encrypted threat: Bitcoin’s social cost and regulatory responses](https://web.archive.org/web/20220107084533/https://www.suerf.org/docx/f_88b3febc5798a734026c82c1012408f5_38771_suerf.pdf) - Jan 2022. A comprehensive study by SUERF - The European Money and Finance Forum that details the net negative effects of bitcoin to society.
* [The Third Web](https://tante.cc/2021/12/17/the-third-web/) - 2021-12-17 - long critical essay including detailed history by [Tante](https://twitter.com/tante)
* [Tante's Web3/NFT FAQ](https://tante.cc/2022/02/09/tantes-blockchain-web3-faq/)
* https://rufuspollock.com/2016/07/02/reflections-on-the-blockchain/ - 2016-07-02 - by Rufus Pollock (mainly a critique of early DAOs and techno-solutionism)
* [Web3 takes trust, too](https://www.bloomberg.com/opinion/articles/2022-01-10/web3-takes-trust-too) - 2022-01-10 by Matt Levine on Bloomberg.com
* [Revolution Now! With Peter Joseph | Bitcoin and Financialization](https://youtu.be/bsghxd1cdeA) - May 21, 2021
* [The Web3 Fraud](https://www.usenix.org/publications/loginonline/web3-fraud) - 2021-12-16 by Nicholas Weaver on usenix.com
* Molly White series - https://blog.mollywhite.net/blockchain/
  * [Blockchain-based systems are not what they say they are](https://blog.mollywhite.net/blockchains-are-not-what-they-say/)
  * [It's not still the early days](https://blog.mollywhite.net/its-not-still-the-early-days/)
  * [Abuse and harassment on the blockchain](https://blog.mollywhite.net/abuse-and-harassment-on-the-blockchain/)
  * [Anonymous cryptocurrency wallets are not so simple](https://blog.mollywhite.net/anonymous-crypto-wallets/)
  * [Cryptocurrency off-ramps, and the pressure towards centralization](https://blog.mollywhite.net/off-ramps/)
  * [Cryptocurrency’s Robinhood effect](https://blog.mollywhite.net/cryptocurrencys-robinhood-effect/)
  * [Abuse on the blockchain – Guest lecture at Stanford University](https://www.youtube.com/watch?v=hXBZ-BXfCSY)
* [Against Web3 and Faux-Decentralization](https://soatok.blog/2021/10/19/against-web3-and-faux-decentralization/) - 2021-10-19 by Soatok
* [The technological case against Bitcoin and blockchain](https://lukeplant.me.uk/blog/posts/the-technological-case-against-bitcoin-and-blockchain/) - 2022-03-05 by Luke Plant
* [The Case Against Crypto](https://www.watershed.co.uk/studio/news/2021/12/03/case-against-crypto) - 2021-12-03 by Martin O'Leary
* [The Case Against Bitcoin](https://bariweiss.substack.com/p/the-case-against-bitcoin?s=r) - 2021-05-14 by Michael W. Green. A portfolio manager discusses the case against bitcoin from a financial and geopolitical perspective.
* [The Register: The dark equation of harm versus good means blockchain’s had its day](https://www.theregister.com/2021/12/06/the_dark_equation_of_harm/) - 2021-12-06
* [Blockchains and Cryptocurrencies: Burn It With Fire](https://www.youtube.com/watch?v=xCHab0dNnj4) - 2018-04-20 by Nicholas Weaver 📺 Nicholas Weaver is a staff researcher with the International Computer Science Institute (ICSI) and lecturer in EECS, where he teaches machine structures and computer security. He earned his Ph.D. in computer science from Berkeley in 2003 and joined ICSI to study network security and measurement. "The entire cryptocurrency and blockchain ecology is rife with frauds, criminalities, and tulip-mania style hype and needs to be properly disposed of into the ashes of history. A “blockchain” is just a horribly inefficient append-only file which costs a literal fortune to secure without actually providing meaningful distributed trust, while cryptocurrencies are provably inferior than actual currencies for legal real world transactions. Beyond the sheer uselessness have emerged a whole host of bad ideas, ranging from the “put a bird^H^H^H^H blockchain on it” hype to unregistered (and mostly fraudulent) securities with “Initial Coin Offerings” to an invitation for massive theft in the form of “smart” contracts."
* [Ross Anderson et al: Bitcoin Redux: crypto crime, and how to tackle it](https://www.lightbluetouchpaper.org/2018/06/01/bitcoin-redux-crypto-crime-and-how-to-tackle-it/) ([full paper](https://weis2018.econinfosec.org/wp-content/uploads/sites/5/2018/05/WEIS_2018_paper_38.pdf))- 2018-06-01 - Anderson is a Professor of Security Engineering at the University Cambridge. Bitcoin Redux explains what’s going wrong in the world of cryptocurrencies. The bitcoin exchanges are developing into a shadow banking system, which do not give their customers actual bitcoin but rather display a "balance" and allow them to transact with others. However if Alice sends Bob a bitcoin, and they’re both customers of the same exchange, it just adjusts their balances rather than doing anything on the blockchain. This is an e-money service, according to European law, but is the law enforced? Not where it matters. We’ve been looking at the details.
  * [Ross Anderson: Why Bitcoin is Not Cash](https://www.youtube.com/watch?v=p9HH_dFcoLc) - 2018-04-10 - 📺 walks through why bitcoin is not cash and the complex legal questions it would need to deal with if it wanted to be.
  * [Ross Anderson: Tracing Stolen Bitcoin](https://www.youtube.com/watch?v=UlLN0QERWBs) - 2018-03-23 - 📺
* [Simon Wardley: A Spoiler for the Future of Bitcoin](https://blog.gardeviance.org/2013/11/a-spoiler-for-future-bitcoin.html) - 2013-11-27 - "As you can guess, I'm not a fan of bitcoin. If left unchecked then I find it has the potential to undermine the importance of Government which is actually not good for competition and not good for the market. I hope none of the above happens and would rather see bitcoin disappear in a puff of history." (NB: he predicts massive appreciation in bitcoin and is concerned how it can undermine government and tax revenue.)
* Kai Stinchcombe series that discusses whether blockchain can solve various real world use-cases better than traditional technologies
  - [Kai Stinchcombe: Ten years in, nobody has come up with a use for blockchain](https://hackernoon.com/ten-years-in-nobody-has-come-up-with-a-use-case-for-blockchain-ee98c180100) - 2017-12-23 - "Each purported use case — from payments to legal documents, from escrow to voting systems—amounts to a set of contortions to add a distributed, encrypted, anonymous ledger where none was needed. What if there isn’t actually any use for a distributed ledger at all? What if, ten years after it was invented, the reason nobody has adopted a distributed ledger at scale is because nobody wants it?"
  - [Kai Stinchcombe: Blockchain is not only crappy technology but a bad vision for the future](https://medium.com/@kaistinchcombe/decentralized-and-trustless-crypto-paradise-is-actually-a-medieval-hellhole-c1ca122efdec) - 2018-05-04 - "Blockchain is not only crappy technology but a bad vision for the future. Its failure to achieve adoption to date is because systems built on trust, norms, and institutions inherently function better than the type of no-need-for-trusted-parties systems blockchain envisions. That’s permanent: no matter how much blockchain improves it is still headed in the wrong direction."
* [Cory Doctorow: When crypto-exchanges go broke, you'll lose it all](https://pluralistic.net/2022/02/03/liquidation-preference/#we-live-in-a-society) - 2022-02-03. Why state backed money is a good thing (a feature not a bug).
  > If you've spent much time around cryptocurrency people, you've probably heard a rant or two about "sound money" and the need to "depoliticize money." This is a foundation of blockchainism: the belief that money is born separate from states, and states invade on the private realm when they "meddle" in the money system.
  >
  > There are at least two serious problems with this ideology. First, it's plain wrong on the historical facts. Money did not emerge from barter systems among people. Money was and is a product of state.
  >
  > But even if you stipulate that money didn't originate among private markets there's another serious historical problem with "sound money." ... It's this: central banks didn't emerge to usurp the private sector's control over money. Central banks were created because without them, finance was subject to wild, terrifying, ruinous boom/bust cycles. What's more, without a central bank, money was subject to naked political meddling, which central banks (sometimes) moderated.
* Internet pioneer/Silicon Valley legend Tim O'Reilly on Web3:
  - [Why it’s too early to get excited about Web3](https://www.oreilly.com/radar/why-its-too-early-to-get-excited-about-web3/) - 2021-12-13
  - ["Get ready for the crash"](https://www.cbsnews.com/news/cryptocurrency-nft-blockchain-web3-tim-oreilly/) - CBS Money Watch - 2022-02-09
  - [Crypto and NFTs are "Pretty Serious Speculative Bubble"](https://decrypt.co/92676/internet-guru-tim-oreilly-crypto-nfts-serious-speculative-bubble) - 2022-02-10
* [David Rosenthal: Can We Mitigate Cryptocurrencies' Externalities?](https://blog.dshr.org/2022/02/ee380-talk.html) - 2022-02-09. Having built a decentralized consensus system using Proof-of-Work (http://dx.doi.org/10.1145/945445.945451) the author has the technical knowledge to explain the design faults and limitations of permissionless blockchain systems, as well as highlighting the economic and environmental issues. Summary of critique:
  > * That the externalities I describe don't exist. You'll have a hard time proving that the waste of electricity and hardware, and the crime wave, are imaginary.
  > * That although the externalities do exist, the benefits of decentralization outweigh them. The problem here is that since the systems are not actually decentralized, we get the externalities but don't get the benefits.
  > * That although the externalities do exist, and the systems aren't dencentralized, they're making so much money that we shouldn't worry. The problem here is that the amount of actual money you can get out of a cryptocurrency equals the amount of actual money that has been put in, minus the actual costs of mining. So the big picture is that although there may be winners, in aggregate the system loses money.
  * [Economies of Scale in Peer-to-Peer Networks](https://blog.dshr.org/2014/10/economies-of-scale-in-peer-to-peer.html) - 2014-10-07. Network effects lead to centralization in p2p (e.g. Bitcoin) and no good way to mitigate this.
* [Charlie Stross: Why I want Bitcoin to Die in Fire](https://www.antipope.org/charlie/blog-static/2013/12/why-i-want-bitcoin-to-die-in-a.html) - 2013-12
* [The Maltese Falcon](https://privatebank.jpmorgan.com/content/dam/jpm-wm-aem/global/pb/en/insights/eye-on-the-market/the-maltese-falcoin.pdf) - critique of bitcoin and financial properties of crypto assets from the CIO of JP Morgan bank. 2021-02-10
* [Vivaldi CEO: Why Vivaldi will never create ThinkCoin](https://vivaldi.com/blog/why-vivaldi-will-never-create-thinkcoin/) - 2022-01-13 - Jon von Tetzchner: “if you look beyond the hype, you’ll find nothing more than a pyramid scheme posing as currency.”
* [Centralizing Control: Why Bitcoin is Dangerous](https://salbayat.org/centralizing-control-why-bitcoin-is-dangerous/) - 2022-04-02 - Sal Bayat: “Democratic governance is fundamentally incompatible with existing cryptocurrency systems as they can only represent the interests of those in control of the system.”

### Economists

* Stephanie Kelton [Cryptocurrency and Fiat Money](https://www.youtube.com/watch?v=84wTEf9Acik) - 2017-12-23
* Richard Thaler [Economics Nobel prize winner, Richard Thaler: “The market that looks most like a bubble to me is Bitcoin and its brethren”](https://econews.pt/2018/01/22/economics-nobel-prize-winner-richard-thaler-the-market-that-looks-most-like-a-bubble-to-me-is-bitcoin-and-its-brethren/) - 2018-01-22
* Various ['Only good for drug dealers': More Nobel prize winners snub bitcoin](https://finance.yahoo.com/news/good-drug-dealers-nobel-prize-winners-snub-bitcoin-184903784.html?ref=hackernoon.com) - 2018-04-27
* Robert Shiller [The Old Allure of New Money](https://www.project-syndicate.org/commentary/cryptocurrencies-scientific-narrative-by-robert-j--shiller-2018-05?barrier=accesspay) - 2018-05-21
* Abhijit Banerjee [Nobel Prize Winning Economist Abhijit Banerjee: Is Blockchain the Key to Financial Inclusion?](https://blockchain.news/news/nobel-prize-winning-economist-abhijit-banerjee-is-blockchain-the-key-to-financial-inclusion) - 2020-01-20
* Steve Keen [Cryptocurrencies, Debt, and the Economy: Steve Keen interviewed by Layne Hartsell](http://www.koreaittimes.com/news/articleView.html?idxno=103792) - 2021-02-17
* Amartya Sen [Prannoy Roy's Townhall With Amartya Sen On Economy, Farm Laws: Full Transcript ](https://www.ndtv.com/india-news/prannoy-roys-townhall-with-amartya-sen-on-indian-economy-farm-laws-full-transcript-2385071) - 2021-03-06
* Jeffrey Sachs [Famed economist Jeffrey Sachs rails against Bitcoin: Highly polluting and ‘almost like counterfeiting’](https://fortune.com/2021/03/16/bitcoin-jeffrey-sachs-critiques-btc/) - 2021-03-16
* Paul Krugman [Technobabble, Libertarian Derp and Bitcoin](https://nytimes.com/2021/05/20/opinion/cryptocurrency-bitcoin.html) - 2021-05-20
* Tyler Cowen [What the Crypto Crowd Doesn't Understand About Economics](https://www.bloomberg.com/opinion/articles/2021-06-21/what-the-crypto-crowd-doesn-t-understand-about-economics) - 2021-06-20
* Yanis Varoufakis [What is money, really? And why Bitcoin is not the answer (even if blockchain is brilliant & potentially helpful in democratising money)](https://www.yanisvaroufakis.eu/2021/08/02/what-is-money/)  - 2021-08-02
* Daron Acemoğlu [The Bitcoin Fountainhead](https://www.project-syndicate.org/commentary/bitcoin-an-appealing-distraction-by-daron-acemoglu-2021-10?barrier=accesspay) - 2021-10-05
* Joseph Stiglitz [Nobel Prize Economist Joseph Stiglitz Calls Regulators to Ban Cryptocurrencies](https://deep-resonance.org/2021/10/28/nobel-prize-economist-joseph-stiglitz-calls-regulators-to-ban-cryptocurrencies/) - 2021-10-28
* Richard Thaler [Economics Nobel prize winner, Richard Thaler: “The market that looks most like a bubble to me is Bitcoin and its brethren”](https://econews.pt/2018/01/22/economics-nobel-prize-winner-richard-thaler-the-market-that-looks-most-like-a-bubble-to-me-is-bitcoin-and-its-brethren/) - 2018-01-22
* Yanis Varoufakis [Yanis Varoufakis on Crypto & the Left, and Techno-Feudalism](https://the-crypto-syllabus.com/yanis-varoufakis-on-techno-feudalism/) - 2022-01-26
* Tyler Cowen [The Crypto Crash Strengthens the Case for Crypto](https://www.bloombergquint.com/gadfly/crypto-crash-strengthens-case-for-crypto-s-long-term-future) - 2022-01-27
* Jesse Frederik [Blockchain, the amazing solution for almost nothing](https://thecorrespondent.com/655/blockchain-the-amazing-solution-for-almost-nothing/84495599980-95473476) - 2020-08-21 - "Blockchain technology is going to change everything: the shipping industry, the financial system, government … in fact, what won’t it change? But enthusiasm for it mainly stems from a lack of knowledge and understanding. The blockchain is a solution in search of a problem."
* [Vice: ‘Crypto Ruined My Life’: The Mental Health Crisis Hitting Bitcoin Investors](https://www.vice.com/en/article/akvn8z/crypto-bad-for-mental-health) - 2022-02-16 - The stress and anxiety that goes with funneling your life savings into a volatile market is no joke.
* [Ed Zitron: Solutions That Create Problems](https://ez.substack.com/p/solutions-that-create-problems) - 2022-02-22 - [The thing about Web3 is that it is uniquely useless. I have actively searched for an explanation as to why it's the future, what products it would allow us to build, what sort of *good* it would provide, and I cannot even at my most optimistic find a real use case](https://twitter.com/edzitron/status/1495891031979704321)

### Ponzi aspect

* [Financial Times: Why bitcoin is worse than a Madoff-style Ponzi scheme](https://web.archive.org/web/20220113183816/https://www.reddit.com/r/CryptoReality/comments/rm78e3/financial_times_why_bitcoin_is_worse_than_a/) - 2021-12-22. A Ponzi scheme is a zero-sum enterprise. But bitcoin is a negative-sum phenomenon that you can’t even pursue a claim against, argues Robert McCauley. [Original](https://ft.com/content/83a14261-598d-4601-87fc-5dde528b33d0)
* [Seattle Times: Bitcoin is basically a Ponzi scheme](https://seattletimes.com/opinion/bitcoin-is-basically-a-ponzi-scheme/) - 2018-01-30
* [Bitcoin is a Ponzi](https://ic.unicamp.br/~stolfi/bitcoin/2020-12-31-bitcoin-ponzi.html) - 2020-12-13 by Prof Jorge Stolfi
* [Financial Times: Albanian lessons for regulators nervously eyeing the crypto world](https://www.ft.com/content/810367e5-e0b1-4221-b303-f3012a177437) - 2021-07-05 - Albania’s 1990s pyramid scheme debacle highlights risks of regulatory paralysis on the cryptocurrency explosion
  > Once upon a time in Albania, a scrappy, alternative finance industry emerged to take on and eventually supplant a sclerotic, technologically-backward banking system. The lessons from its dramatic collapse remain relevant today. 
  >
  > Essentially, what was initially touted as a post-communist entrepreneurial success story proved to be pyramid schemes of breathtaking proportions. Slick marketing and lofty promises turned an informal, decentralised, crime-facilitating ecosystem into a mainstream mania that sucked in multitudes of people, unchecked by feeble and fitful regulatory warnings.
* [Jacobin: Cryptocurrency Is a Giant Ponzi Scheme](https://jacobinmag.com/2022/01/cryptocurrency-scam-blockchain-bitcoin-economy-decentralization) - 2022-01-21

### Crypto and energy consumption

* [Bitcoin Energy Consumption Index](https://digiconomist.net/bitcoin-energy-consumption)
* [Why Bitcoin Is Bad For The Environment](https://newyorker.com/news/daily-comment/why-bitcoin-is-bad-for-the-environment) - 2021-04-22
* [Energy power usage CryptoArt, ETH, Blockchain spreadsheet](https://docs.google.com/spreadsheets/d/1hzzxMbytOZ1mYl9kLh_SvM6kne6JI_mdCfHIoNapr5M/edit#gid=0)
* [How Do We Solve Bitcoin's Energy Problem?](https://www.theguardian.com/technology/2022/jan/30/how-do-we-solve-bitcoins-carbon-problem) - 2022-01-30

### Scams/frauds

* [People Building ‘Blockchain City’ in Wyoming Scammed by Hackers - Vice](https://www.vice.com/en/article/k7w3am/people-building-blockchain-city-in-wyoming-scammed-by-hackers) - 2022-01-12 - On Monday, CityDAO—the group that bought 40 acres of Wyoming in hopes of "building a city on the Ethereum blockchain”—announced that its Discord server was hacked and members' funds were successfully stolen as a result.
* [Web3 is going just great](https://web3isgoinggreat.com/) - A timeline of scams related to cryptocurrencies, NFTs, and web3 projects since the beginning of 2021 by Molly White

### DAOs

* [Is The DAO going to be DOA?](https://steemit.com/crypto-news/@dan/is-the-dao-going-to-be-doa) - 2016-05-16 - by Dan Larimer (founder of BitShares and much else). Larimer sets out most of the basic critiques of DAOs as governance innovation extremely well:
  > Fancy technology can obscure our assessment of what is really going on. The DAO solves a single problem: the corrupt trustee or administrator. It replaces voluntary compliance with a corporation’s charter under threat of lawsuit, with automated compliance with software defined rules. This subtle change may be enough to bypass regulatory hurdles facing traditional trustee’s and administrators, but it doesn’t solve most of the problems the regulations were attempting to address.
  >
  > What The DAO doesn’t solve is all of the other problems inherent with any joint venture. These are people problems, economic problems, and political problems. In some sense, The DAO creates many new problems caused by its ridged rules and expensive machine-enforced process for change.
  >
  > The DAO doesn’t solve the “group trap” where by losers subsidize winners. It disempowers the individual actor and forces him to submit to group decision making. It doesn’t make raising money cheaper for companies, it just adds blockchain-enforced bureaucratic and political processes.
* [DAOs and the nature of human collaboration](https://world.hey.com/marin/daos-and-the-nature-of-human-collaboration-be162918) - 2021-08-12 by Marin Petrov. A critique of DAOs and technosolutionism.

### NFTs

Non-fungible tokens.

* [OpenSea, Web3, and Aggregation Theory](https://stratechery.com/2022/opensea-raises-money-bans-nfts-openseas-value-cryptos-aggregators/) - 2022-01-05 - Ben Thompson of Stratechery
* [Brian Eno on NFTs & Automaticism](https://the-crypto-syllabus.com/brian-eno-on-nfts-and-automatism/)
* [Detailed twitter thread by @NFTEthics alleging fraudulent or close to fraudulent behaviour by a major NTF influencer named BeanieMaxi ](https://twitter.com/NFTethics/status/1483051289022017538) - 2022-01-17 ([cached](./assets/Thread by @NFTethics re beaniemaxi.pdf))
* [Jacobin: NFTs Are, Quite Simply, Bullshit](https://jacobinmag.com/2022/01/nfts-fallon-paris-hilton-bored-ape-digital-imagery-commodification) - 2022-01-26

### Specific use cases

* Event ticketing: [NFT tickets — a realistic look at a big trend](https://medium.com/@ticketpark/nft-tickets-a-realistic-look-at-a-big-trend-ae813d6f885d) – 2021-12-14
* NFT games: [“Play-to-earn” and Bullshit Jobs](https://paulbutler.org/2021/play-to-earn-and-bullshit-jobs/) - December 28, 2021 by Paul Butler - An interesting reflexion linking web3's "Play-to-earn" concept to David Graeber's [Bullshit Jobs](https://en.wikipedia.org/wiki/Bullshit_Jobs)
* NFT games: [Crypto Games: Report from hell](https://www.youtube.com/watch?v=YHz0xpU5Tu8) - Good video reviewing and discussing crypto games

### Humour

* [Crypto Curious](https://www.youtube.com/watch?v=N8f-BQFo7lw) - South Park on NFTs - 2021-12-21
* [N-FT: Non-Functioning Tower](https://www.nonfunctioningtower.com) - NFT satire - 2022-03-07
* [“a normal person explains cryptocurrency”](https://twitter.com/avalonpenrose/status/1473753174787772418) by Avalon Penrose - 2021-12-22
* [“my crypto friend calls me every day and this is what he sound like”](https://www.youtube.com/watch?v=TUB9jgMuC7U) by Flula - 2021-02-22
* [The Billion-Dollar Bitcoin Scam](https://www.youtube.com/watch?v=YCuGpfMSmck) - Ordinary Things - 2020-05-31 - “What is Bitcoin? Is Bitcoin a scam? And how did Bitcoin become what it is today? Who was the Dread Pirate Roberts and what happened to the Silk Road?”
* [Cryptocurrencies: Last Week Tonight with John Oliver (HBO)](https://www.youtube.com/watch?v=g6iDZspbRMg) - 2018-03-12
* [Don’t Understand Bitcoin? This Man Will Mumble An Explanation At You](https://www.youtube.com/watch?v=4APcgsRdW6w) by ClickHole - 2015-07-7
* [If Cryptocurrency was Honest](https://www.youtube.com/watch?v=GUs5y9leCyA)
* [If NFTs were Honest](https://www.youtube.com/watch?v=sG_v4bb2e4k)

### Twitter users

Whilst these users may not solely discuss crypto or web3, they do discuss it regularly, and have consistently provided well-written critique.

* https://twitter.com/web3isgreat
* https://twitter.com/ncweaver
* https://twitter.com/molly0xFFF
* https://twitter.com/smdiehl
  - [Crypto Criticism Threads](https://gist.github.com/sdiehl/7706ef44d951a2025fd658d1dd8687af)
* https://twitter.com/rufuspollock
* https://twitter.com/troll_lock
* https://twitter.com/CasPiancey -"Under promise, under deliver" co-host @cryptocriticpod *opinions are mine, not my employer* odds and ends @protos hold no crypto or crypto stonks
* https://twitter.com/BennettTomlin - I do data science and track down frauds | 74% backed | Co-host @CryptoCriticPod | Writing @fud_letter | Discord: https://discord.gg/YpAUqNkhSC
* https://twitter.com/SilvermanJacob (staff writer New Republic) & https://twitter.com/ben_mckenzie - "apparently I now write about crypto"
* https://twitter.com/doctorow

### Tether, and other stablecoins


notes/news.md
# News

## [[notes/celsius|Celsius]] sued by former partner claiming lack of payment

https://twitter.com/0x_b1/status/1545153652624691200 - thread from Jason Stone on twitter

https://www.bloomberg.com/news/articles/2022-07-07/embattled-crypto-lender-celsius-accused-of-fraud-by-ex-employee

> Behind the scenes, Celsius was investing customer funds in risky trading strategies, without proper controls, according to the lawsuit. Starting in August 2020, Celsius started transferring hundreds of millions of dollars to Stone’s company, KeyFi. 
> 
> Though the two firms didn’t have a written agreement, Stone was given the private cryptographic keys to the funds, meaning he could have run off with them, according to the complaint.
> 
> Stone was tasked with investing the money through DeFi. Short for “decentralized finance,” it’s a constellation of apps that let users borrow, lend and trade with each other, without middlemen. 
> 
> At the time, many of the apps were paying users huge rewards in proprietary cryptocurrencies. KeyFi earned more than $800 million for Celsius through DeFi strategies, according to the lawsuit. Stone says Celsius was supposed to pay him a 20% share of most of that but never did.
> 
> The problem, according to Stone, was that Celsius mainly took deposits in Bitcoin and Ethereum, but his strategies earned rewards in other coins. That meant that if Bitcoin and Ethereum went up faster than the others, Celsius could end up owing more than it had, even if it was earning money. 
> 
> Celsius also kept track of customers’ deposits in U.S. dollar terms, even if they were actually owed Bitcoin or other tokens, according to the suit. When this error was discovered, it resulted in “a $100-$200 million hole on its balance sheet,” Stone alleged.
> 
> Celsius raised $50 million in 2018 by selling a proprietary token, CEL, and the company and its executives held large stocks of the cryptocurrency. 
> 
> Stone now claims that Celsius in 2020 used $90 million worth of Bitcoin deposits to “artificially inflate” the price of CEL. He says that let Mashinsky “enrich himself,” and enabled Celsius to borrow against its CEL holdings. He also says that Celsius borrowed 1 billion Tethers from the stablecoin issuer to cover the hole in its balance sheet.
> 
> Stone says he ended his relationship with Celsius in March 2021 once he discovered the improprieties. In a thread on twitter, Stone wrote that Celsius “assured me they had risk management and hedging in place. But in late Feb 2021, we discovered Celsius had lied to us.”

Fascinating part of this is the background on the DeFi trading strategies and especially the claim by KeyFi that they aren't responsible for hedging out "impermanent loss". However, as we have noted before impermanent loss is anything but impermanent and is the key risk hole in liquidity strategies. Being able to trade and not be responsible for impermanent loss is like selling naked puts into a bull market and not being responsible for downside risk if the market sours. It's a heads I win, tails you lose strategy. Will be interested to see how this lawsuit goes.

This is someone points out in the twitter comments:

https://twitter.com/mgnr_io/status/1545287527422099456

## [[notes/voyager-digital|Voyager Digital]] Files for [[concepts/chapter-11|Chapter 11]] - 2022-07-06

From  https://www.ledgerinsights.com/crypto-firm-voyager-digital-files-for-chapter-11/

> Voyager Digital, the Toronto-listed crypto firm that lent bankrupt Three Arrows Capital $650 million, filed for Chapter 11 protection in New York bankruptcy court yesterday. At the end of the quarter to March 31, it said it had 1.2 million funded accounts.
>
> Court filings show that its assets are between $1 billion and $10 billion, as are its liabilities. A balance sheet wasn’t filed, so the true deficit is currently unclear. (Update: a document shows its outstanding crypto loans amount to $1.1 billion.) Voyager requested to extend the deadline for sharing financial details. It had $5.8 billion in assets on the platform at the end of the first quarter.
>
> The company says it currently has $1.3 billion in crypto-assets on its platform plus $110 million in crypto-assets and cash owned by the company. Additionally, $350 million of client cash is segregated in a client account and should be returned in full to customers, subject to compliance requirements.

Editorial:

- One interesting aspect of chapter 11 filing is that we are now getting more and more insight into the relatively opaque world of crypto finance.
- These crypto funds are using traditional state-based processes e.g. chapter 11

Coverage

- https://www.ledgerinsights.com/crypto-firm-voyager-digital-files-for-chapter-11/
- https://www.theguardian.com/technology/2022/jul/06/cryptocurrency-broker-voyager-digital-files-for-bankruptcy-protection

## 2022-07-02 - Fall-out from [Three Arrows Capital](3ac-three-arrows-capital.md) as BlockFi sells and Voyager faces bankruptcy

Crypto-lender [[notes/blockfi|BlockFi]] are a creditor of collapsed crypto hedge fund [Three Arrows Capital](3ac-three-arrows-capital.md). After taking $350m at a $3bn valuation in March 2021 it looks like they may be worth only 1-10% on the dollar with an option to be acquired for up to $240m by FTX

https://decrypt.co/104286/blockfi-ftx-credit-deal-acquisition-terms

https://decrypt.co/104200/blockfi-ceo-denies-ftx-cquire-crypto-lender-25m

Meanwhile Voyager Digital who are owed $650m by [Three Arrows Capital](3ac-three-arrows-capital.md) has halted all withdrawals and is likely going bankrupt. 

https://decrypt.co/104294/voyager-digital-halts-withdrawals-three-arrows-capital-default

## May 17: Famous investor Bill Ackman weighs in on [Terra/Luna collapse](terra-crash-may-2022.md):

https://twitter.com/BillAckman/status/1526398201003130880


site/pages/notes.js
import Link from "next/link";
import { allOtherPages } from "contentlayer/generated"
import { YOUTUBE_ID_REGEX } from "lib/constants";

export async function getStaticProps() {
  const posts = allOtherPages
    .filter((page) => page._raw.sourceFileDir === "notes")
    .map((page) => {
      return {
        title: page.title || null,
        description: page.description || null,
        date: page.date,
        image: page.image || null,
        youtube: page.youtube || null,
        link: `/${page._raw.flattenedPath}`,
      };
    })
    .sort((a, b) => new Date(b.date) - new Date(a.date));

  return { props: { posts } };
}

export default function Notes({ posts }) {
  return (
    <div className="relative pt-16 pb-20 px-4 sm:px-6 lg:pt-24 lg:pb-28 lg:px-8">
      <div className="relative max-w-7xl mx-auto">
        <div className="text-center">
          <p className="mt-3 max-w-2xl mx-auto text-3xl text-gray-500 sm:mt-4">
            Our latest articles and explorations.
          </p>
        </div>
        <div className="mt-12 max-w-lg mx-auto grid gap-6 lg:grid-cols-3 lg:max-w-none">
          {posts && posts.map(({ title, description, image, youtube, link }) => (
            title && description && 
              (<div key={link} className="flex flex-col rounded-lg shadow-lg overflow-hidden">
                <div className={`bg-stone-100 flex-shrink-0 ${image ? "h-48" : "h-10"}`}>
                  {image && <img className="" width="100%" height="100%" src={image} alt={title} />}
                </div>
                <div className="flex-1 bg-slate-800 p-6 flex flex-col justify-between">
                  <div className="flex-1">
                    <p className="text-sm font-medium text-indigo-600 dark:text-yellow-500">
                      {youtube && YOUTUBE_ID_REGEX.test(youtube)
                        ? (<a target="_blank" rel="noopener" href={youtube} className="hover:underline">
                          Video
                        </a>) 
                        : <p>Article</p>
                      }
                    </p>
                    <Link href={link}>
                      <a className="block mt-2">
                        <p className="text-xl font-semibold text-slate-900 dark:text-slate-300">{title}</p>
                        <p className="mt-3 text-base text-gray-500">
                          {description && description.length > 150
                            ? `${description.slice(0, image ? 120 : 200)} ...`
                            : description
                          }
                        </p>
                      </a>
                    </Link>
                  </div>
                </div>
              </div>)
          ))}
        </div>
      </div>
    </div>
  );
}